### ISSUE TITLE
convergence

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [ss3sim/estgrowth#3]
Convergence issues with no age data (stats shown for internal estimation):
- 70% of the runs with no age data and only fishery length data are not converging
- 63% of the runs with no age data and fishery and survey length data are not converging

The convergence issue goes away when the model is supplemented with conditional age-at-length data. Also less (35% and 25%) convergence issues with external estimation. Also the convergence is worse for the two-way trip rather than fishing at Fmsy.

**Comment by iantaylor-NOAA** [comment]
I think that in a real-world application, if you didn't have age data or
really distinct modes showing cohorts in the length comps, you would rarely
try to estimate growth internally. Or maybe you would put some priors on
growth based on similar species or something like that.

I know it makes for less clean results to compare both the bias of
parameter estimates and the convergence rates, but I think that 70% of runs
not converging isn't so much an issue as it is a result worthy of writing
up in your manuscript.

**Comment by juanlvalero** [comment]
Indeed, also in the internal estimation scenarios we were estimating all
parameters, we also have pending to run scenarios where we only estimated
the CVs, but fix the other growth parameters at external estimates.
Juan
